### PR TITLE
destroy swiper instance when component element is destroied

### PR DESCRIPTION
**Pull request by Matt-Jensen**
Fix memory leak by destroying the Swiper instance when the component's element will be destroyed.  Also makes use of ember-cli-shims, simplifies swiper-slide component, and removes unnecessary DOM query.

### CODE REVIEW

File: tests/integration/components/swiper-container-test.js [hunk 1]
```diff
@@ -66,3 +67,23 @@ test('on initialization, does not call `afterSwiperInit` if `afterSwiperInit` is
   this.render(hbs`{{#swiper-container}} Foo {{/swiper-container}}`);
   assert.equal(spy.callCount, 0);
 });
+
+test('it destroies the Swiper instance when the component element destroied', function(assert) {
```
Comment (bmx269) [inline tests/integration/components/swiper-container-test.js]: "destroyed" is the proper spelling for the past tense of "destroy"

**Comment by GCheung55** [comment]
Anything else needed to get this merged?

**Comment by topaxi** [comment]
I believe the ember-cli-shims modules are outdated. According to https://github.com/emberjs/rfcs/blob/master/text/0176-javascript-module-api.md and https://github.com/ember-cli/ember-modules-codemod they look quite different now.

**Comment by Matt-Jensen** [comment]
@topaxi your right, I'll get that updated.

**Comment by Matt-Jensen** [comment]
@Suven ready for 0.6.3 release

**Pull request merged**